core/common/src/main/java/alluxio/client/file/PrestoContext.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.client.file;
+
+import alluxio.client.quota.CacheQuota;
+import alluxio.client.quota.CacheScope;
+import alluxio.wire.FileInfo;
+
+import com.google.common.base.MoreObjects;
+
+import java.util.Objects;
+
+/**
+ * Context when running in Presto.
+ */
+public class PrestoContext {
```

Comment:
CacheContext?

Reply:
After second thoughts, I agree `CacheContext` can be better,